src/contrib/cluster/Akka.Cluster.Tools/PublishSubscribe/DistributedMessages.cs

```diff
@@ -661,51 +667,21 @@ public override int GetHashCode()
         /// <returns>TBD</returns>
         public override string ToString()
         {
-            return string.Format("SendToAll<path:{0}, excludeSelf:{1}, message:{2}>", Path, ExcludeSelf, Message);
+            return $"SendToAll<path:{Path}, excludeSelf:{ExcludeSelf}, message:{Message}>";
         }
     }
 
     /// <summary>
     /// TBD
     /// </summary>
     [Serializable]
-    public sealed class GetTopics : IEquatable<GetTopics>
```

Comment:
This is serializable class. Singleton messages are not guaranteed to be deserialized back into `Instance` object. Moreover, usually they are guaranteed to be different objects.